src/Compiler/Service/ServiceParseTreeWalk.fs

```diff
@@ -445,7 +445,7 @@ module SyntaxTraversal =
                 | SynExpr.AnonRecd(copyInfo = copyOpt; recordFields = fields) ->
                     [
                         match copyOpt with
-                        | Some(expr, blockSep) ->
+                        | Some(expr, Some blockSep) ->
```

Comment:
The change is the same for two kinds of record expressions. I think extracting something like `diveIntoRecordLikeExpr` would make it easier to maintain it down the road.

Alternatively, we could later unify the two kinds of record expressions into a single `SynExpr` case, like we do for computation expression specific expressions and their normal counterparts.